### PR TITLE
Added references to the new Splunk-App-Section

### PR DESCRIPTION
**Pull request by EParzefall**
References to the new Splunk App for ownCloud give it more visibility in the auditing and monitoring context.
Backport to 10.7 necessary.

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/logging/admin_audit.adoc [hunk 1]
```diff
@@ -19,6 +20,8 @@ The {oc-marketplace-url}/apps/admin_audit[Auditing] app is an Enterprise only ap
 * Enabling / disabling of ownCloud Apps
 * Executions of OCC commands (CLI)
 
+TIP: You may also want to check out the {splunk-url}[_ownCloud App for Splunk_]. For more information, read this xref:pages/configuration/integration/splunk.adoc[section].
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/logging/admin_audit.adoc:23]:
```suggestion
TIP: You may also want to check out the {splunk-url}[_ownCloud App for Splunk_]. For more information, read this xref:configuration/integration/splunk.adoc[section].
```

File: modules/admin_manual/pages/enterprise/reporting/metrics.adoc [hunk 2]
```diff
@@ -15,7 +16,9 @@ visualizations and to set alerts for certain events of interest. They can be per
 ownCloud Audit Logs (provided by the {oc-marketplace-url}/apps/admin_audit[Auditing App]) to gather time
 series data and to create a reporting engine for ownCloud.
 
-Specifically, the extension adds:
+If you want to use Splunk in addition, check out xref:pages/configuration/integration/splunk.adoc[ownCloud App for Splunk].
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/reporting/metrics.adoc:19]:
```suggestion
If you want to use Splunk in addition, check out xref:configuration/integration/splunk.adoc[ownCloud App for Splunk].
```

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
some small things, just commit the suggestions any you can merge it

**Comment by phil-davis** [comment]
Backport 10.7 #3523

**Pull request merged**